pkg/service/utgen/assets/test_generation.toml

```diff
@@ -168,9 +253,79 @@ new_tests:
     npm install assert
     {%- endif %}
   test_tags: happy path
-    ...
+
+refactored_source_code: |
+  # Here is the modified source code that retains original functionality but improves testability.
+  ...
 ```
 
+additions:
+  additional_instructions_text: |
```

Comment:
we already have something in the name of additional_instructions_text , might create confusion , also instead of sending all languages for all ,let's find out the language and send text accordingly